src/App.tsx

```diff
@@ -1,61 +1,115 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { Todo } from './Types/Todo';
+import { User } from './Types/User';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const findUserById = (userId: number) => (
+  usersFromServer.find(user => user.id === userId) as User
+);
+
+const allTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: findUserById(todo.userId),
+}));
+
+export const App: React.FC<{}> = () => {
+  const [title, setTitle] = useState('');
+  const [currUser, selectUser] = useState('');
+  const [todos, setTodos] = useState(allTodos);
+  const [errorTitle, setErrorTitle] = useState(false);
+  const [errorUser, setErrorUser] = useState(false);
```

Comment:
booleans should be names starting with a verb like 'has' or 'is'

const hasTitleError
const hasUserError